pkg/test/kube/accessor.go

```diff
@@ -19,7 +19,7 @@ import (
 	"strings"
 	"time"
 
-	"github.com/hashicorp/go-multierror"
+	multierror "github.com/hashicorp/go-multierror"
```

Comment:
I assume this is some sanity fix, if so, it is ok. Otherwise consider remove?

Reply:
Done.